README.md

```diff
@@ -39,6 +39,31 @@ jobs:
         branch: ${{ github.ref }}
 ```
 
+An example workflow to authenticate with GitHub Platform via Deploy Keys or in general SSH:
+
+```yaml
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        ssh-key: ${{ secrets.SSH_PRIVATE_KEY }}
```

Comment:
I'll also suggest highlighting that the `persist-credentials` input should be set to `true`

Reply:
Sure, that should not result in a problem.

Reply:
Done